### PR TITLE
VideoPress: handle how the VideoCard looks when no data is provided

### LINKED ISSUE #26113: VideoPress: Reduce flicker when rendering video thumbnail
The app shows flickers in the process of rendering the video cards.  Find a way to remove the visual jumps. Even more, consider implementing video placeholders.

### PR DESCRIPTION
**Pull request by retrofox**
This PR introduces the `isBlank` property to the `<VideoCard />` component. When True, the component hides its whole content but doesn't lose the dimms. 
Also, when thumbnail property is not defined, the component shows a Video Icon.

Fixes #

#### Changes proposed in this Pull Request:

*

#### Testing instructions:

* Go to StoryBook
* Use the tools to check the 

https://user-images.githubusercontent.com/77539/189202185-ed7df3b0-dfca-4ccf-8193-5a0283eaaa84.mov

size and show/hide the placeholder when there is no thumbnail

### CODE REVIEW

File: projects/packages/videopress/src/client/admin/components/video-card/index.tsx [hunk 1]
```diff
@@ -121,7 +121,15 @@ export const VideoThumbnail: React.FC< VideoThumbnailProps & VideoThumbnailDropd
 				</div>
 			) }
 
-			<img src={ thumbnail } alt={ __( 'Video thumbnail', 'jetpack-videopress-pkg' ) } />
+			{ thumbnail && (
+				<img src={ thumbnail } alt={ __( 'Video thumbnail', 'jetpack-videopress-pkg' ) } />
+			) }
```
Comment (renatoagds) [inline projects/packages/videopress/src/client/admin/components/video-card/index.tsx]: Any reason to not use ternary here?

```
thumbnail ? ... : ...
```

File: projects/packages/videopress/src/client/admin/components/video-card/types.ts [hunk 2]
```diff
@@ -58,6 +58,16 @@ export type VideoDetailsProps = {
 };
 
 export type VideoCardProps = VideoPressVideo & {
+	/**
+	 * Whether the card should look "blank" style.
+	 */
+	isBlank?: boolean;
```
Comment (renatoagds) [inline projects/packages/videopress/src/client/admin/components/video-card/types.ts:64]: Which is the use case for `isBlank`? Don't get where we'll use it

File: projects/packages/videopress/src/client/admin/components/video-card/index.tsx (hunk 1 above)
Comment (retrofox) [reply projects/packages/videopress/src/client/admin/components/video-card/index.tsx]: good call. I changed the conditions again and again and didn't realize about the ternary, which is a better way. Updating...

File: projects/packages/videopress/src/client/admin/components/video-card/types.ts (hunk 2 above)
Comment (retrofox) [reply projects/packages/videopress/src/client/admin/components/video-card/types.ts:64]: It's initial kick to implement something not clearly defined in the design. However, this is a pretty typical case when pulling data from the client. We can keep polishing and tweaking the UI.

File: projects/packages/videopress/src/client/admin/components/video-card/index.tsx (hunk 1 above)
Comment (retrofox) [reply projects/packages/videopress/src/client/admin/components/video-card/index.tsx]: https://github.com/Automattic/jetpack/pull/26124/commits/e9c2f36bf37f49a1d65a8af15e1953d51cef9fcd thanks

File: projects/packages/videopress/src/client/admin/components/video-card/types.ts (hunk 2 above)
Comment (oskosk) [reply projects/packages/videopress/src/client/admin/components/video-card/types.ts:64]: Can you please clarify what's that something not clearly defined in the design?

Without further details, it looks to me that the dimension behavior (and the thumbnail replacement) should be the regular behavior of the component without needing an extra prop.

Comment (retrofox) [reply projects/packages/videopress/src/client/admin/components/video-card/types.ts:64]: > Can you please clarify what's that something not clearly defined in the design?

Yes, it's what you mentioned below. "the dimension behavior (and the thumbnail replacement)" is not mentioned explicitly in the design, unless I'm missing something.

Probably _isBlank_ property doesn't make sense, though. Let's keep the card dimension invariable independently whether the data is provided or not.

Comment (retrofox) [reply projects/packages/videopress/src/client/admin/components/video-card/types.ts:64]: I've removed the `isBlank` property, but the card will keep the dimension even when no data is provided https://github.com/Automattic/jetpack/pull/26124/commits/a1ee7a1841af92fd9b53eb3580c4b06667b39641

Comment (oskosk) [reply projects/packages/videopress/src/client/admin/components/video-card/types.ts:64]: excellent!

Comment (retrofox) [reply projects/packages/videopress/src/client/admin/components/video-card/types.ts:64]: thanks both 🙇‍♂️

**Review by oskosk — APPROVED** [review]
LGTM!

**Pull request merged**